cost-analyzer/values.yaml

```diff
@@ -2525,10 +2525,41 @@ kubecostAggregator:
   # extraEnv: can be used to add new environment variables to the aggregator pod
 
   # the below settings should only be modified with support from Kubecost staff
+
+  # How many threads the read database is configured with (i.e. Kubecost API /
+  # UI queries). If increasing this value, it is recommended to increase the
+  # aggregator's memory requests & limits.
+  # default: 1
   dbReadThreads: 1
+  # How many threads the write database is configured with (i.e. ingestion of
+  # new data from S3). If increasing this value, it is recommended to increase
+  # the aggregator's memory requests & limits.
+  # default: 1
   dbWriteThreads: 1
+  # How many threads to use when ingesting Asset/Allocation/CloudCost data
+  # from the federated store bucket. In most cases the default is sufficient,
+  # but can be increased if trying to backfill historical data.
+  # default: 1
   dbConcurrentIngestionCount: 1
-  dbCopyFull: false
+  # dbCopyFull: "true" can improve the time it takes to copy the write DB,
+  # at the expense of additional memory usages.
+  dbCopyFull: "false"
```

Comment:
You changed this from boolean to string. Was that intentional? Does it work with the current type of string? Haven't inspected the templates where this is referenced.

Reply:
env vars should be strings, but helpers quotes it, so changing back to bool